content/tutorials/rust/hello-world.md

```diff
@@ -175,7 +187,7 @@ fn main() {
     let args: Vec<String> = std::env::args().collect();
     let args_slice: Vec<&str> = args.iter().map(|s| &**s).collect();
 
-    let result = match &*args_slice {
+    let _ = match &*args_slice {
```

Comment:
The `result` value was being used below to print errors if any occurred

Reply:
My bad... I missed that part of code during copy/paste. I'll revert.

Reply:
The 2nd section did not have it... hence the problem.